src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,7 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+1. Configure off-cluster object storage
```

Comment:
might want to add more detail here, or just integrate https://github.com/deis/workflow/pull/141